website_coupon/controllers/main.py

```diff
@@ -1,25 +1,26 @@
 # -*- coding: utf-8 -*-
 
-from datetime import datetime
-from dateutil import parser
-from odoo import http, tools, _
+from odoo import http
 from odoo.http import request
 
 
 class WebsiteCoupon(http.Controller):
 
     @http.route(['/shop/cart'], type='http', auth="public", website=True)
     def cart(self, **post):
-        """This function is overwritten because we need to pass the value 'coupon_not_available'
-        to the template, inorder to show the error message to the user that, 'this coupon is not available'. """
+        """ This function is overwritten because we need to pass the value
+        'coupon_not_available' to the template, in order to show the error
+        message to the user that, 'this coupon is not available'.
+        """
 
         order = request.website.sale_get_order()
         if order:
             from_currency = order.company_id.currency_id
             to_currency = order.pricelist_id.currency_id
-            compute_currency = lambda price: from_currency.compute(price, to_currency)
+            compute_currency = lambda price: from_currency.compute(  # noqa
+                price, to_currency)
         else:
-            compute_currency = lambda price: price
+            compute_currency = lambda price: price  # noqa
```

Comment:
why noqa here ?

Reply:
Just because flake8 prefers a true function definition instead of assigning a lambda to a variable (which is no more a lambda then). Anyway you are right, I will strip that flake8 stuff out, it has nothing to do here.